doc/howtos/rdtraining/setup.rst

```diff
@@ -0,0 +1,408 @@
+is not installed through pip and must be installed manually in version 0.12.5 for it to support
+headers and footers.
+
+.. code-block:: console
+
+    $ cd /tmp/
+    $ sudo wget https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/wkhtmltox_0.12.5-1.focal_amd64.deb
+    $ sudo gdebi --n wkhtmltox_0.12.5-1.focal_amd64.deb
+    $ sudo ln -s /usr/local/bin/wkhtmltopdf /usr/bin
+    $ sudo ln -s /usr/local/bin/wkhtmltoimage /usr/bin
+
+Right-to-left interface support
+-------------------------------
+
+.. code-block:: console
+
+    $ sudo apt-get install nodejs npm
+    $ sudo npm install -g rtlcss
+
+Install PostgreSQL
+------------------
+
+Odoo needs a PostgreSQL server to run properly.
+
+.. code-block:: console
+
+    $ sudo apt install postgresql postgresql-client
+    $ sudo -u postgres createuser -s $USER
+
+
+Some sql useful commands :
+
+.. code-block:: console
+
+    $ createdb $DB_NAME #Create a database
+    $ dropdb $DB_NAME #Drop a database
+
+    $ psql $DB_NAME #Connect into a database
+        \l #List all the available databases
+        \dt #List all the table of the $DB_NAME database
+        \d $TABLE_NAME #Show the structure of the table $TABLE_NAME
+        \q #Quit the psql environment (ctrl + d)
+
+Run the server
+==============
+
+Running odoo-bin
+----------------
+
+Once all dependencies are set up, Odoo can be launched by running odoo-bin, the command-line interface of the server.
+
+.. code-block:: console
+
+    $ cd src/odoo/
+    $ ./odoo-bin --addons-path="addons/,../enterprise/" -d rd-demo
+
+There are multiples :ref:`command-line arguments <reference/cmdline/server>` that you can use to
+configure the server. In this training, you will only need some of them.
+
+* `-d <database>`
+  The database that is going to be used.
+
+* `--addons-path <directories>`
+  Comma-separated list of directories in which modules are stored. These directories are scanned for
+  modules.
+
+* `--limit-time-cpu <limit>`
+  Prevents the worker from using more than <limit> CPU seconds for each request.
+
+* `--limit-time-real <limit>`
+  Prevents the worker from taking longer than <limit> seconds to process a request.
+
+The last two are going to be used to avoid killing the worker when debugging the source code.
+
+.. tip:: If when you start the server you have an error like `AttributeError: module '$MODULE_NAME' has no attribute '$ATTRIBUTE'`
+
+         You may need to re-install the module with `$ pip install --upgrade --force-reinstall $MODULE_NAME`
+
+         If this error occurs with more than one module you may need to re-install all the
+         requirements with `$ pip3 install --upgrade --force-reinstall -r requirements.txt`
+
+         You can also remove the python cache to solve the issue
+
+         .. code-block:: console
+
+            $ cd /home/$USER/.local/lib/python3.8/site-packages/
+            $ find -name '*.pyc' -type f -delete
```

Comment:
Did you face this issue? Seems weird... or it means you faced another error previously.